examples/tests/valid/if_else.w

```diff
@@ -0,0 +1,16 @@
+bring cloud;
```

Comment:
Should be enough to have the `if_elif_else` test, shouldn't it?

Reply:
I separated it so I could test each step of the if, but I can put it in a single file in the future